tests/rot_test.cpp

```diff
@@ -0,0 +1,53 @@
+#include <stdlib.h>
+#include <memory>
+#include <string>
+
+#include "catch/catch.hpp"
+#include "calendar.h"
+#include "item.h"
+#include "cata_utility.h"
+#include "game.h"
+#include "flat_set.h"
+#include "point.h"
+
+
+static bool is_nearly( float value, float expected )
```

Comment:
The catch2 test framework we're using already provides [support for approximate comparisons](https://github.com/catchorg/Catch2/blob/master/docs/assertions.md#floating-point-comparisons); you shouldn't need to implement it yourself.